src/01_create_parameter_logistic_model_LAAL.R

```diff
@@ -0,0 +1,168 @@
+library(data.table) 
+
+source("src/dimorphism_model_class.R")
+source("src/calculator_ROC_class.R")
+source("src/evaluate_model_function.R")
+source("src/get_prediction_sex_plot_function.R")
+source("src/get_sex_probability_plot_function.R")
+source("src/regretion_to_data_frame_coefficients_function.R")
+
+directorioTDP <- ("data/raw/")
+nombreArchivoCSV <- file.path(directorioTDP,"morfometria_albatros-laysan_guadalupe.csv")
+nombreArchivoJSON <- file.path(directorioTDP,"datapackage.json")
+rutaResultados = ('data/processed/')
+
+Metadatos <- jsonlite::fromJSON(nombreArchivoJSON)
+Datos <- data.table(read.csv(nombreArchivoCSV))
+n_datos <- nrow(Datos)
+
+proporcion_entrenamiento <- 0.80
+proporcion_validacion <- 1 - proporcion_entrenamiento
+
+variablesParaModelo <- c("Longitud_Craneo", "Longitud_Pico", "Ancho_Craneo", "Altura_Pico", "Tarso", "Longitud_AlaCerrada", "Longitud_AlaAbierta", "Envergadura")
+nombre_columnas <- c("(Intercept)", variablesParaModelo)
+n_repeticiones <- 10
+#2000
```

Comment:
Por favor borra este comentario

Reply:
Hola, Nepo! Gracias por las correciones :) 
Me gustaría por el momento dejar ese comentario porque en un futuro debo cambiar `n_repeticiones <- 10` por `n_repeticiones <- 2000` (la versión original). Además, este PR es para cambiar todo **(excepto por los sripts)**, así que creo que la rebeldía de dejarlo igual está justificada jaja

Reply:
jajajaja Perfecto me parece muy bien. 
> **C5:** Commented-Out Code
> It makes me crazy to see stretches of code that are commented out. Who knows how old it is? Who knows whether or not it’s meaningful? Yet no one will delete it because everyone assumes someone else needs it or has plans for it.
> That code sits there and rots, getting less and less relevant with every passing day. It calls functions that no longer exist. It uses variables whose names have changed. It follows conventions that are long obsolete. It pollutes the modules that contain it and distracts the people who try to read it. Commented-out code is an abomination.
> When you see commented-out code, delete it! Don’t worry, the source code control system still remembers it. If anyone really needs it, he or she can go back and check out a previous version. Don’t suffer commented-out code to survive.